packs/st2/actions/kv_get.py

```diff
@@ -13,7 +13,7 @@ def run(self, key, decompress=False):
         _key = self.client.keys.get_by_name(key)
 
         if not _key:
-            return None
+            raise Exception("Key does not exist")
```

Comment:
There are also some other `kv.get*` actions which we should also update for consistency (`kv_get_object.py`).

Also, this will affect all the existing work-flows which use this action when the key doesn't exist - previously it went to the "on-success" branch, now it will go into "on-error".

As @manasdk noted, this could break thing so at the very least we should add a changelog entry or something.